samples/openapi3/client/petstore/python-experimental/tests/test_deserialization.py

```diff
@@ -125,4 +125,16 @@ def test_deserialize_mammal(self):
         deserialized = self.deserialize(response, (petstore_api.Mammal,), True)
         self.assertTrue(isinstance(deserialized, petstore_api.Zebra))
         self.assertEqual(deserialized.type, zebra_type)
-        self.assertEqual(deserialized.class_name, class_name)
\ No newline at end of file
+        self.assertEqual(deserialized.class_name, class_name)
+
+    def test_deserialize_fruit(self):
+        """
+        deserialize fruit
+        fruitReq is a oneOf composed schema model with discriminator, including 'null' type.
+        """
+
+        # Unmarshal 'null' value
+        data = None
+        response = MockResponse(data=json.dumps(data))
+        deserialized = self.deserialize(response, (petstore_api.FruitReq, type(None)), True)
+        self.assertEqual(type(deserialized), type(None))
```

Comment:
I don't think that this will work anymore.
How about [here](https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/python/python-experimental/model_utils.mustache#L66)
adding:
```
oneof_anyof_classes = cls._composed_schemas.get('oneOf', ()) + cls._composed_schemas.get('anyOf', ())
if oneof_anyof_classes and none_type in oneof_anyof_classes and args[0] is None:
  return None
```
Then you can also delete the oneof_anyof_classes definition from lines 78-80.

Reply:
Was this done; where? In the PR I don't see any code added in the __new__ method.

Reply:
> Where was this done? In the PR I don't see any code added in the `__new__` method.

Where was what done?

Reply:
My above suggestion asked that you add code in the new method. So I am asking where the requested code was added.

Reply:
ok, I missed that proposed change.

Reply:
Is it expected that your proposed change has no check for "allOf"?

Reply:
Can you describe a payload and schema that would need this allOf check where the check is missing?

when deserializing None where none is in OneOf or AnyOf, any allOf definition of object property fields can never be met by a None payload because None is not a pojo.

Further down in the new function we cover the use caas where we have allof definitionsl, verify that the payload conforms to it then deserialize to an anyof or oneof pojo payload.

Reply:
That was a question for you. You wrote this code, so I am asking you.

Reply:
Does my above answer meet your needs? I don't understand when we would need to check allof but we miss that check. In the answer I describe both cases that do and don't check allof and why.

Reply:
Thanks. I just copied your code as is without checking the correctness.